### ISSUE TITLE
Crash When Parsing Unions in Tuples

### ISSUE THREAD

**Issue by NeilKleistGao** [NeilKleistGao/ts-to-mls#32]
```ts
function s2(t: [boolean, string | number]): string | number {
    if (t[0]) return t[1]
    else return 0
}
```

**Comment by NeilKleistGao** [comment]
```
[info] - Tuple *** FAILED ***
[info]   scala.scalajs.js.JavaScriptException: TypeError: Cannot read properties of undefined (reading 'symbol')
[info]   at ts2mls.TSTypeObject.symbol$lzycompute(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35454:50)                         
[info]   at ts2mls.TSTypeObject.symbol(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35579:65)                                    
[info]   at ts2mls.TSTypeObject.declaration$lzycompute(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35538:15)                    
[info]   at ts2mls.TSTypeObject.declaration(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35612:68)                               
[info]   at ts2mls.TSSourceFile.getObjectType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29537:18)                             
[info]   at ts2mls.TSSourceFile.getStructuralType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29700:63)                         
[info]   at ts2mls.TSSourceFile.getObjectType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29546:14)                             
[info]   at ts2mls.TSSourceFile.getTupleElements(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29724:16)                          
[info]   at ts2mls.TSSourceFile.getObjectType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29502:50)                             
[info]   at ts2mls.TSSourceFile.getFunctionParametersType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29615:16)
[info]   ...                                                                                                                                               
[info] - Tuple Convert *** FAILED ***                                                                                                                      
[info]   scala.scalajs.js.JavaScriptException: TypeError: Cannot read properties of undefined (reading 'symbol')                                           
[info]   at ts2mls.TSTypeObject.symbol$lzycompute(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35454:50)                         
[info]   at ts2mls.TSTypeObject.symbol(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35579:65)                                    
[info]   at ts2mls.TSTypeObject.declaration$lzycompute(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35538:15)                    
[info]   at ts2mls.TSTypeObject.declaration(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:35612:68)                               
[info]   at ts2mls.TSSourceFile.getObjectType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29537:18)                             
[info]   at ts2mls.TSSourceFile.getStructuralType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29700:63)                         
[info]   at ts2mls.TSSourceFile.getObjectType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29546:14)                             
[info]   at ts2mls.TSSourceFile.getTupleElements(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29724:16)                          
[info]   at ts2mls.TSSourceFile.getObjectType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29502:50)                             
[info]   at ts2mls.TSSourceFile.getFunctionParametersType(/mnt/d/Code/ts-to-mls/target/scala-2.13/ts-to-mls-test-fastopt/main.js:29615:16)                 
[info]   ...  
```

**Comment by NeilKleistGao** [comment]
Fixed in #34
